Sources/Rendering/Core/Camera/index.js

```diff
@@ -417,6 +417,10 @@ function vtkCamera(publicAPI, model) {
     // invert to get hmd -> world
     mat4.invert(viewMatrix, viewMatrix);
 
```

Comment:
need to save viewMatrix to model...

Reply:
Nope, that's intentional.
`viewMatrix` is defined in the vtkCamera function. 
See https://github.com/sankhesh/vtk-js/blob/00c21d6f77aa9c82cd97e5053fa2508c1eb9912e/Sources/Rendering/Core/Camera/index.js#L29